kumascript/macros/CSSSyntax.ejs

```diff
@@ -1,442 +1,324 @@
-    "de"   : "Mögliche Werte: ",
-    "ja"   : "使用可能な値: ",
-    "ru"   : "Возможные значения: "
-});
-
-let s_number_followed_by = mdn.localString({
-    "en-US": "a number followed by",
-    "de"   : "eine Nummer gefolgt von",
-});
-
-let s_like = mdn.localString({
-    "en-US": "like",
-    "de"   : "wie",
-    "ru"   : "как"
-});
-
-let typeInfo = {
-    "angle": {
-        "title": s_possible_values + s_number_followed_by + "'deg', 'grad', 'rad' " + s_or + " 'turn', " + s_like + " 2turn, 1.3rad, -60deg " + s_or + " 0grad."
-    },
-    "blend-mode": {
-        "title": s_possible_values + "normal, multiply, screen, overlay, darken, lighten, color-dodge, color-burn, hard-light, soft-light, difference, exclusion, hue, saturation, color, luminosity"
-    },
-    "color": {
-        "typeLinkName": "color_value"
-    },
-    "custom-ident": {},
-    "filter-function": {
-        "title": s_possible_values + "blur(), brightness(), contrast(), drop-shadow(), grayscale(), hue-rotate(), invert(), opacity(), saturate(), sepia()"
-    },
-    "flex": {
-        "typeLinkName": "flex_value"
-    },
-    "frequency": {},
-    "ident": {},
-    "image": {},
-    "integer": {},
-    "length": {
-        "title": s_possible_values + s_number_followed_by + "'em', 'ex', 'ch', 'rem', 'px', 'cm', 'mm', 'in', 'vh', 'vw', 'vmin', 'vmax', 'pt', 'pc' " + s_or + " 'px', " + s_like + " 3px, 1.5cm, -0.5em " + s_or + " 0"
-    },
-    "number": {},
-    "percentage": {},
-    "position": {
-        "typeLinkName": "position_value"
-    },
-    "resolution": {},
-    "single-transition-timing-function": {
-        "title": s_possible_values + "cubic-bezier(), steps(), linear, ease, ease-in, ease-out, east-in-out, step-start-step-end"
-    },
-    "string": {},
-    "time": {
-        "title": s_possible_values + s_number_followed_by + "'s' " + s_or + " 'ms', " + s_like + " 3s, -2.5ms " + s_or + " 0s."
-    },
-    "timing-function": {
-        "title": s_possible_values + "cubic-bezier(), steps(), linear, ease, ease-in, ease-out, east-in-out, step-start-step-end"
-    },
-    "url": {},
-    "uri": {},
-    "transform-function": {
-        "title": s_possible_values + "matrix(), matrix3d(), rotate(), rotate3d(), rotateX(), rotateY(), rotateZ(), scale(), scale3d(), scaleX(), scaleY(), scaleZ(), skewX(), skewY(), translate(), translate3d(), translateX(), translateY(), translateZ()"
+}
+
+// get the contents of webref
+const parsedWebRef = await webRefData.listAll();
+
+// get all the value syntaxes
+let valuespaces = {};
+for (const spec of Object.values(parsedWebRef)) {
+  valuespaces = {...valuespaces, ...spec.valuespaces};
+}
+
+// get the property name from the page slug
+const propertyName = $0 || env.slug.split('/').pop().toLowerCase();
+
+/**
+ * Get the formal syntax for a property from the webref data, given:
+ * `propertyName`: the name of the property
+ * `parsedWebRef`: the webref data
+ */
+function getPropertySyntax(propertyName, parsedWebRef) {
```

Comment:
Afaik no need to pass `parsedWebRef`, because it should be available here from the outer scope:

```suggestion
function getPropertySyntax(propertyName) {
```